src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultSecretManagerProvider.cs

```diff
@@ -69,9 +69,9 @@ internal ISecretsRepository CreateSecretsRepository()
             else if (secretStorageType != null && secretStorageType.Equals("keyvault", StringComparison.OrdinalIgnoreCase))
             {
                 string azureWebJobsSecretStorageKeyVaultName = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsSecretStorageKeyVaultName);
-                string azureWebJobsSecretStorageKeyVaultConnectionString = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsSecretStorageKeyVaultConnectionString);
+                string azureWebJobsSecretStorageKeyVaultUserManagedClientId = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsSecretStorageKeyVaultUserManagedClientId);
```

Comment:
I think we could reasonably name this `azureWebJobsSecretStorageKeyVaultClientId` just to simplify slightly.

Reply:
Agree, the shorter the better :)